src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -457,7 +340,6 @@ of `pan-x | pan-y | manipulation` you prevent the default double-tap
 behavior.
 
 This allows you to implement a double-tap gesture yourself.
-In the case of IE10+, it also eliminates the 300ms click delay.
 
 Below is a list of the available parameters for *touch-action*.
```

Comment:
Should I drop this table completely and just add a note on `touch-action: pan-y pinch-zoom` or just drop and leave it out?